app/views/planning_applications/review/assessment_details/_form.html.erb

```diff
@@ -1,94 +1,116 @@
-<%= form_with(
-      model: @form,
-      url: planning_application_review_assessment_details_path(planning_application),
-      method: :put
-    ) do |form| %>
-  <%= form.govuk_error_summary %>
-  <% planning_application.application_type.assessment_details.each_with_index do |assessment_detail, index| %>
-    <% next if assessment_detail == "check_publicity" %>
-    <% unless index == 0 %>
-      <hr class="govuk-section-break govuk-section-break--visible govuk-section-break--m">
-    <% end %>
-    <%= form.govuk_radio_buttons_fieldset(
-          :"#{assessment_detail}_reviewer_verdict",
-          legend: {text: t(".#{assessment_detail}")}
-        ) do %>
-      <%= render(
-            StatusTags::Reviewing::AssessmentDetailComponent.new(
-              planning_application: planning_application,
-              assessment_detail: form.object.send(assessment_detail)
-            )
-          ) %>
-      <% if assessment_detail == "site_description" %>
-        <p class="govuk-body govuk-!-margin-top-2"><%= govuk_link_to "View site on Google Maps", map_link(@planning_application.full_address), new_tab: true %></p>
-      <% end %>
-      <% if assessment_detail == "consultation_summary" && @consultation&.consultees&.any? %>
+<%= bops_task_accordion do |accordion| %>
```

Comment:
Could we leave this as just being a form and move the `bops_task_accordion` in the `app/views/planning_applications/review/tasks/_review_assessment.html.erb`?

Reply:
I think if this isn't _only_ a single form we should avoid using the `_form.html.erb` conventional name for it, since `render "form"` won't behave in the same way as the Rails default arrangement.

Reply:
Yeah I'll move all this into "review_assessment_summaries"

Reply:
Moved the accordion logic to there now which makes more sense 👍